paths/intro-to-github/14-videorecap.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Video Recap
+permalink: /intro-to-github/14/
+sidebar:
+  nav: "intro-nav"
+main-content: |
+  ## Lightning Video
```

Comment:
@hollenberry should we add a little more context here? Perhaps something like: "We know it may seem like a lot to remember, so we created a short video of just the steps required to complete the activity."